pkg/debug/constants/consts.go

```diff
@@ -0,0 +1,5 @@
+package constants
+
+const (
+	DebugGQLPlaygroundEndpoint = "/_zot/debug/graphql-playground"
```

Comment:
Would split it as 
Debug = "/_zot/debug
and gqlPlayground = Debug + "/graphql..." etc

Reply:
Modified as requested